exercise/if-exists-then.js

```diff
@@ -0,0 +1,52 @@
+/*
+## If Exists Then
+
+### Instructions
+
+Create three functions that receive an array and a function each:
+
+- `Every` that returns true if every element of the array respects the
+condition of the received function and false otherwise.
+- `Some` that returns true if at least one element of the array respects the
+condition of the received function and false otherwise.
+- `None` that returns true if none of the elements of the array respects the
+condition of the received function and false otherwise.
+
+The use of `[].every` and `[].some` is forbidden for this exercise.
+
+### Notions
+
+- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
+- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every
+*/
+
```

Comment:
Use this
```js
Array.prototype.some = Array.prototype.every = undefined
```
Instead of checking in the code for the string which can be easly tricked
```js
const k = 'e'+'very'
[][k] ...
```